### DISCUSSION TITLE
How to fit when there are level jumps in your data?

### DISCUSSION THREAD

**Discussion by blazespinnaker** [Nixtla/statsforecast#393] [Q&A]
In the time series that I'm trying to fit on, there are a number of well known level jumps occurring.   Eg, at the beginning of the new year, the underlying data is being changed to be based on new fundamentals so the december -> january isnt necessarily relevant.

Is there known approach for handling these with models such as ARIMA?

**Comment by AzulGarza** [comment]
hey @blazespinnaker! One approach would be to build an external regressor, let's say `jump`, equal to 1 when there is an expected or historical jump and zero otherwise. And then use `AutoARIMA` with that external regressor. [Here](https://nixtla.github.io/statsforecast/examples/exogenous.html)'s an example.
